### ISSUE TITLE
Use blue as the accent color in light mode

### ISSUE THREAD

**Issue by mxmou** [ScratchAddons/ScratchAddons#7358]
### Why this would be helpful

The orange accent color used on the settings page and in the popup doesn't have enough contrast with white.

### My suggestion

In #4983, I suggested using orange for selection/focus and blue for emphasis. Because orange on white and blue on dark gray don't have 3:1 contrast, I think it's better to use orange everywhere in dark mode and blue everywhere in light mode.

### Possible alternatives

#6027

**Comment by WorldLanguages** [comment]
> The orange accent color used on the settings page and in the popup doesn't have enough contrast with white.

I can only think of the headers, the popup tabs and highlighted buttons. Most other UI elements don't mix orange with white, right? Or am I missing something?

**Comment by mxmou** [comment]
> > The orange accent color used on the settings page and in the popup doesn't have enough contrast with white.
> 
> I can only think of the headers, the popup tabs and highlighted buttons. Most other UI elements don't mix orange with white, right? Or am I missing something?

This issue is about orange elements on white, for example the selected category indicator and enabled addon switches.
